Rationale.md

```diff
@@ -0,0 +1,64 @@
+# Rationale
+
+Prettier is an opinionated JavaScript formatter and this document gives a rationale behind those opinions.
+
+## What is prettier concerned about
```

Comment:
I would say the first thing in this section should be about consistency, because that's really what this project is about. The other things are goals centered around that, but ultimately it's about this:

```
### Consistency

Prettier exists for one purpose: to enforce consistency across your entire project. Not only do we output code with consistent whitespace, prettier will lay out code according to a wrapping algorithm based on a maximum line width. That means that long expressions will be broken up across lines, removing the need for manual layout from the programmer which inevitably leads to inconsistency.
```